translations/translation_sv.xml

```diff
@@ -4,19 +4,18 @@
     <!-- SV translation By piplarsson -->
 		<text name="CP_activated" 																		text="Aktiverad" />
 		<text name="CP_deactivated" 																	text="Avaktiverad" />
-		<text name="CP_automatic" 																		text="auto" />
```

Comment:
why remove ?

Reply:
I'm sorry, I did the translation from an earlier version.  I can of course redo the latest if needed.  otherwise you may be able to add the missing line

Reply:
correct translation of that line is "automatisk"